exercises/concept/freelancer-rates/.docs/instructions.md

```diff
@@ -22,24 +16,28 @@ dayRate(89);
 
 The day rate does not need to be rounded or changed to a "fixed" precision.
 
-## 2. Calculate the month rate, given an hourly rate and a discount
+## 2. Calculate the number of workdays given a fixed budget and hourly rate
 
-Implement a function to calculate the month rate, and apply a discount:
+Another day, a project manager offers the freelancer to work on a project with a fixed budget.
+Given the fixed budget and the freelancer's hourly rate, help him calculate the number of days the project would take to complete
 
 ```javascript
-monthRate(89, 0.42);
-// => 9086
+CHANGE THIS EXAMPLE>>>>
+daysInBudget(20000, 89);    
+// => 28
 ```
 
-The discount is always passed as a number, where `42%` becomes `0.42`. The result _must_ be rounded up to the nearest whole number.
+The result _must_ be rounded down to the nearest whole number.
```

Comment:
Sidenote: To address the issue that a lot of people missed the rounding part in this exercise, I would recommend changing the emphasis here:
```suggestion
The result must be **rounded down** to the nearest whole number.
```
Same applies to task 3.